apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateTextQuestion.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './CreateQuestion.scss';
+import { Control, Controller } from 'react-hook-form';
+import { Input } from '../../../../components/FormInputs/Input';
+import { SelectInput } from '../../../../components/FormInputs/SelectInput';
+import {
+    CreateCodedQuestionRequest,
+    CreateDateQuestionRequest,
+    CreateNumericQuestionRequest,
+    CreateTextQuestionRequest,
+    MessagingInfo,
+    ReportingInfo
+} from '../../generated';
+import { QuestionFormType } from './CreateQuestion';
+
+type CreateQuestionFormType = CreateNumericQuestionRequest &
+    CreateCodedQuestionRequest &
+    CreateDateQuestionRequest &
+    CreateTextQuestionRequest &
+    ReportingInfo &
+    MessagingInfo &
+    QuestionFormType;
+type TextQuestionProps = {
+    control?: Control<CreateQuestionFormType, any>;
+};
+export const CreateTextQuestion = ({ control }: TextQuestionProps) => {
+    return (
+        <>
+            <Controller
+                control={control}
+                name="mask"
+                rules={{ required: { value: true, message: 'Mask required' } }}
+                render={({ field: { onChange, value }, fieldState: { error } }) => (
+                    <SelectInput
+                        defaultValue={value}
+                        label="Mask"
+                        onChange={onChange}
+                        options={[
+                            {
+                                name: 'Text',
+                                value: 'text'
+                            }
+                        ]}
+                        error={error?.message}
+                        required
+                    />
+                )}
+            />
+            <Controller
+                control={control}
+                name="fieldLength"
+                rules={{
+                    required: { value: true, message: 'Field length required' },
+                    maxLength: 3
+                }}
+                render={({ field: { onChange, value }, fieldState: { error } }) => (
+                    <Input
+                        onChange={onChange}
+                        className="field-space"
+                        value={value}
+                        label="Field length"
+                        type="number"
+                        error={error?.message}
+                        required
+                    />
+                )}
+            />
+        </>
```

Comment:
Issues (blocking):
1. Unable to enter data in the `Field length` field.
2. Valid `mask` options are not listed. the valid mask values for a text question are available on the mask enum in the CreateTextQuestionRequest.ts). The displayed options for the mask entries can come from Figma directly
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/442758fd-9d27-4b60-a4e1-c85637873f3e)